homeassistant/components/recorder.py

```diff
@@ -102,14 +102,13 @@ def setup(hass, config):
     """Setup the recorder."""
     # pylint: disable=global-statement
     global _INSTANCE
-
-    _INSTANCE = Recorder(hass)
+    _INSTANCE = Recorder(hass, config.get('history', {}))
```

Comment:
Can you add a `CONFIG_SCHEMA` constant to this component to validate and coerce your newly added config option. See [group component](https://github.com/balloob/home-assistant/blob/dev/homeassistant/components/group.py#L61-L63) for an example.